modules/nw-udn-cr.adoc

```diff
@@ -8,11 +8,10 @@
 
 The following procedure creates a `UserDefinedNetwork` CR that is namespace scoped. Based upon your use case, create your request by using either the `my-layer-two-udn.yaml` example for a `Layer2` topology type or the `my-layer-three-udn.yaml` example for a `Layer3` topology type.
 
-//We won't have these pieces till GA in 4.18.
-//[NOTE]
-//====
-//If any cluster default networked pods exist before the user-defined network is created, any further pods created in this namespace will return an error message: `What_is_this`?
-//====
+.Perquisites
+
+* You have logged in with `cluster-admin` privileges, or you have `view` and `edit` role-based access control (RBAC).
```

Comment:
Wouldn't saying "you have `view` and `edit` role" imply users they should configure it manually?
How about "or project-admin (self-provisioner) privileges?

We have added cluster-roles with aggregation to `view` and `edit` roles with the required permissions for UDN CRs https://github.com/openshift/openshift-docs/pull/92273  
Once a non-admin user logs in they should be granted with view & edit roles automatically and have the require permissions to work with UDN CRs.

Reply:
@ormergi we are consistent with that language use `You have` across the docs repo, and I don't believe it implies that you have to manually configure it.

Reply:
Sounds better, it just that any logged in user will have view and edit RBAC.